src/modules/toolbox.rs

```diff
@@ -0,0 +1,69 @@
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::toolbox::ToolboxConfig;
+use crate::formatter::StringFormatter;
+
+/// Creates a module with the current toolbox image
+///
+/// Will display the toolbox image if `$toolbox_NAME` is set.
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let toolbox_env = context.get_env("TOOLBOX_PATH")?;
```

Comment:
While `TOOLBOX_PATH` is set in every toolbx container, it does not hold information important for this particular module. It is a path to the internal Toolbx binary used for container initialization. Instead I'd look for the file `/run/.toolboxenv`. It's presence signifies that the container is in fact a toolbx container.

Reply:
Thanks, yes it's better. I'll work on that.